xbmc/guilib/GUIMoverControl.dox

```diff
@@ -37,10 +43,42 @@ important, as xml tags are case-sensitive.
 |----------------:|:--------------------------------------------------------------|
 | texturefocus    | Specifies the image file which should be displayed when the mover has focus. [See here for additional information about textures](http://kodi.wiki/view/Texture_Attributes).
 | texturenofocus  | Specifies the image file which should be displayed when the mover does not have focus.
+| movingspeed     | Specifies the allowed directional movements and respective configurations. The <c>`<movingspeed>`</c> tag section can contain as many <c>`<eventconfig>`</c> tags as required.
 
 
 --------------------------------------------------------------------------------
-\section Mover_Control_sect3 See also
+\section Resize_Control_sect3 Note on use of movingspeed tag
+
```

Comment:
Missing `@skinning_v20` revision tag somewhere

Reply:
add a revision section as done in ranges control